packages/report-server/src/reportBuilder/transform/functions/mergeRows/mergeRows.ts

```diff
@@ -13,19 +13,23 @@ import { buildCreateGroupKey } from './createGroupKey';
 import { buildGetMergeStrategy } from './getMergeStrategy';
 import { starSingleOrMultipleColumnsValidator } from '../transformValidators';
 
-type GroupRowsParams = {
+type MergeRowsParams = {
   createGroupKey: (row: Row) => string;
   getMergeStrategy: (field: string) => keyof typeof mergeStrategies;
   where: (parser: TransformParser) => boolean;
 };
 
 const paramsValidator = yup.object().shape({
-  by: starSingleOrMultipleColumnsValidator,
-  mergeUsing: yup.lazy((value: unknown) => {
-    const mergeStrategyNameValidator = yup
+  groupBy: starSingleOrMultipleColumnsValidator,
+  using: yup.lazy((value: unknown) => {
+    const undefinedMergeStrategyNameValidator = yup
```

Comment:
Would slightly prefer `optionalMerge...` because of the `.required()` use below (clever, btw)

Reply:
I was kinda annoyed by this tbh, as far as I can tell `yup` has no `undefined` validator, so instead I returned a non-required string validator. Seems a bit funky to me but it gets the job done 🤷 

Yup, `optional` is a better term, will rename 👍